_posts/2022-10-13-DPGA-membership.md

```diff
@@ -18,12 +18,12 @@ We're thrilled to be participating in this mission. We believe our work to enabl
 
 The Foundation for Public Code’s work aligns with that of the DPGA in the following ways, which are included as part of the [DPGA’s Annual Roadmap](https://digitalpublicgoods.net/roadmap/):
 
-* we steward the Standard for Public Code, which is [also a Digital Public Good](https://blog.publiccode.net/news/2021/07/28/digital-public-goods.html). Early standard compliance helps important projects scale and achieve results sooner.
+* we steward the Standard for Public Code, which is [also a Digital Public Good]({% post_url 21-07-28-digital-public-goods %}). Early standard compliance helps important projects scale and achieve results sooner.
```

Comment:
```suggestion
* we steward the Standard for Public Code, which is [also a Digital Public Good]({% post_url 2021-07-28-digital-public-goods %}). Early standard compliance helps important projects scale and achieve results sooner.
```
While it seems to work anyway, I think it would be clearer to use the filename as it is.